flask_cors/__init__.py

```diff
@@ -150,6 +153,7 @@ def wrapped_function(*args, **kwargs):
             options.update(_get_app_kwarg_dict())
             options.update(_options)
             _serialize_options(options)
+            logger.debug('Final CORS options: {}'.format(str(options)))

```

Comment:
My vote is to move this logging statement to Cors. init_app. This will get logged on every invocation of the route otherwise. And for the decorator, there is not actually any sense of finality, since this computed at request invocation.